.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -3,6 +3,3 @@ contact_links:
   - name: ⁉️ Need help with Node.js?
     url: https://github.com/nodejs/help/issues/
     about: File an issue in our help repo.
-  - name: 📗 Node.js API Docs
-    url: https://github.com/nodejs/nodejs.org/issues/new/choose
```

Comment:
This should be pointed to nodejs/node repo not this one, since that is where the docs actually live.

Reply:
Thank you, and I've fixed that by offering a direct link to the API Docs of Nodejs itself.